spec/plans/results.fmf

```diff
@@ -0,0 +1,118 @@
+summary: Define format of on-disk storage of results
+title: Results format
+order: 90
+
+description: |
+    The following text defines a YAML file structure tmt uses for storing
+    results. tmt itself will use it when saving results of ``execute`` step,
+    and custom test result are required to follow it when creating their
+    ``results.yaml`` file.
+
+    Results are saved as a YAML file, containing a single list of mappings,
+    one mapping describing a single test result.
+
+    .. code-block::
+
+       # String, name of the test.
+       name: /test/one
+
+       # String, outcome of the test execution.
+       #
+       # pass
+       #     Test execution successfully finished and passed.
+       # info
+       #     Test finished but only produced an informational
+       #     message. Represents a soft pass, used for skipped
+       #     tests and for tests with the :ref:`/spec/tests/result`
+       #     attribute set to *ignore*. Automation must treat
+       #     this as a passed test.
+       # warn
+       #     A problem appeared during test execution which does
+       #     not affect test results but might be worth checking
+       #     and fixing. For example test cleanup phase failed.
+       #     Automation must treat this as a failed test.
+       # error
+       #     Undefined problem encountered during test execution.
+       #     Human inspection is needed to investigate whether it
+       #     was a test bug, infrastructure error or a real test
+       #     failure. Automation must treat it as a failed test.
+       # fail
+       #     Test execution successfully finished and failed.
+       result: "pass"|"fail"|"info"|"warn"|"error"
+
+       # String, optional comment to report with the result.
+       note: "Things were great."
+
+       # List of strings, paths to file logs, relative to $TMT_TEST_DATA path.
```

Comment:
Relative to the TMT_TEST_DATA is expected for custom results file.

For the 'tmt' results.yaml logs are relative to the execute workdir (eg `data/guest/default-0/script-00-1/output.txt`)

Reply:
Added a section WRT `logs` below the code block, explaining the paths, in a8424505. Instead of "relative to execute workdir", I opted for "relative to `results.yaml`", which seems to be equivalent to me.